tests/CloudSearchDomain/CloudSearchDomainTest.php

```diff
@@ -45,5 +45,6 @@ public function testConvertGetToPost()
         $this->assertEquals('application/x-www-form-urlencoded', $request->getHeaderLine('Content-Type'));
         $this->assertEquals(7, $request->getHeaderLine('Content-Length'));
         $this->assertEquals('foo=bar', $request->getBody());
+        $this->assertEquals('', $request->getUri()->getQuery());
```

Comment:
Maybe use `assertSame` otherwise things like `0` and `null` pass this assertion.